test/service/test_manifest.py

```diff
@@ -1358,36 +1358,50 @@ def test_manifest_content_disposition_header(self):
         bundle_fqid = self.bundle_fqid(uuid='f79257a7-dfc6-46d6-ae00-ba4b25313c10',
                                        version='2018-09-14T13:33:14.453337Z')
         self._index_canned_bundle(bundle_fqid)
-        with patch.object(manifest_service, 'datetime') as mock_datetime:
-            mock_datetime.now.return_value = datetime(1985, 10, 25, 1, 21)
-            for format in [ManifestFormat.compact]:
-                for filters, expected_name in [
-                    # For a single project, the content disposition file name should
-                    # be the project name followed by the date and time
-                    (
-                        {'project': {'is': ['Single of human pancreas']}},
-                        'Single of human pancreas 1985-10-25 01.21'
-                    ),
-                    # In all other cases, the standard content disposition file name
-                    # should be "hca-manifest-" followed by the manifest key,
-                    # a pair of deterministically derived v5 UUIDs.
-                    (
-                        {'project': {'is': ['Single of human pancreas', 'Mouse Melanoma']}},
-                        'hca-manifest-20d97863-d8cf-54f3-8575-0f9593d3d7ef.4bc67e84-4873-591f-b524-a5fe4ec215eb'
-                    ),
-                    (
-                        {},
-                        'hca-manifest-c3cf398e-1927-5aae-ba2a-81d8d1800b2d.4bc67e84-4873-591f-b524-a5fe4ec215eb'
-                    )
-                ]:
-                    with self.subTest(filters=filters, format=format):
-                        manifest, num_partitions = self._get_manifest_object(format, filters)
-                        self.assertFalse(manifest.was_cached)
-                        self.assertEqual(1, num_partitions)
-                        url = furl(self._service.get_manifest_url(manifest))
-                        expected_cd = f'attachment;filename="{expected_name}.tsv"'
-                        actual_cd = url.args['response-content-disposition']
-                        self.assertEqual(expected_cd, actual_cd)
+        for bundle_notifications_enabled in [True, False]:
```

Comment:
This test would probably benefit from using patch decorators instead of the patch CM.